packages/costa/src/runnable.ts

```diff
@@ -209,12 +214,11 @@ export class PluginRunnable {
   /**
    * Do send handshake message to runnable client, and wait for response.
    */
-  private async handshake(): Promise<boolean> {
-    await this.sendAndWait(PluginOperator.START, {
+  private async handshake(): Promise<PluginMessage> {
```

Comment:
Could you please keep this to return boolean?

Reply:
I think the logic is clearer through exception handling.

Reply:
OK, I got u, Updated.